precompiles/outposts/stride/tx_test.go

```diff
@@ -38,30 +43,76 @@ func (s *PrecompileTestSuite) TestLiquidStake() {
 			},
 			200000,
 			true,
-			fmt.Sprintf(cmn.ErrInvalidNumberOfArgs, 4, 0),
+			fmt.Sprintf(cmn.ErrInvalidNumberOfArgs, 1, 0),
+		},
+		{
+			"fail - token not found",
+			func() []interface{} {
+				err := s.network.App.StakingKeeper.SetParams(s.network.GetContext(), stakingtypes.DefaultParams())
```

Comment:
why is this necessary?

Reply:
to bypass an error and set the bond denom to `stake` so an certain error can occur.